concordium-node/README.md

```diff
@@ -1,26 +1,30 @@
 # Concordium node implementation
+
 ## Dependencies to build the project
-* Rust (stable 1.53 for using static libraries)
-* binutils >= 2.22
-  * For macOS one should use the binutils provided by Xcode.
-* cmake >= 3.8.0
-* [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html)
-  v2.0.0 is what we currently use. Either build from the v2.0.0 tag of the repository using CMake and copy the `flatc` binary somewhere in your PATH, or download a released binary from https://github.com/google/flatbuffers/releases/tag/v2.0.0 and place it somewhere in your PATH.
-* protobuf >= 3.7.1
-* LLVM and Clang >= 3.9
-* PostGreSQL >= 10
+
+- Rust (stable 1.53 for using static libraries)
+- binutils >= 2.22
+  - For macOS one should use the binutils provided by Xcode.
+- cmake >= 3.8.0
+- [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html)
+  v2.0.0 is what we currently use. Either build from the v2.0.0 tag of the repository using CMake and copy the `flatc` binary somewhere in your PATH, or download a released binary from <https://github.com/google/flatbuffers/releases/tag/v2.0.0> and place it somewhere in your PATH.
+- protobuf >= 3.7.1
+- LLVM and Clang >= 3.9
+- PostGreSQL >= 10
 
 ### Optional dependencies
-* [Haskell stack](https://docs.haskellstack.org/en/stable/README/) if **not** building using static libraries
+
+- [Haskell stack](https://docs.haskellstack.org/en/stable/README/) if **not** building using static libraries
 
 ## Supported features
-* instrumentation - switches the default internal counter implementation out with prometheus
-* instrumentation - enables stats data exporting to [prometheus](https://crates.io/crates/prometheus)
-* network_dump - makes the network dumping capabilites available.
-* static - build against static haskell libraries (Linux only)
-* profiling - build against haskell libraries with profiling support enabled (Linux only)
-* collector - enables the build of the node-collector and backend
-* dedup_benchmarks - enable support in the benchmarks for deduplication queues
+
+- instrumentation - switches the default internal counter implementation out with prometheus
+- instrumentation - enables stats data exporting to [prometheus](https://crates.io/crates/prometheus)
+- network_dump - makes the network dumping capabilites available.
+- static - build against static haskell libraries (Linux only)
+- profiling - build against haskell libraries with profiling support enabled (Linux only)
+- collector - enables the build of the node-collector and backend
+- dedup_benchmarks - enable support in the benchmarks for deduplication queues
```

Comment:
I do not see what is changed here. Please refrain from these kinds of changes. I don't see what is improved here at all.

Reply:
It's a markdown linter fix, the symbol for the list items should be. consistent in a file, either. `-` or `*`